### DISCUSSION TITLE
Issues deploying to Vercel with sveltekit

### DISCUSSION THREAD

**Discussion by bmccorm2** [prisma/prisma#14338] [Q&A] [answered]
Hi - I'm having issues deploying a sveltekit project to vercel with prisma.  I can have a simple get endpoint like this and it will work fine.
```
export const get: Load = () => {
	return {
		status: 200,
		body: {
			message: 'all good!'
		}
	};
};
```

However as soon as i include prisma i get errors on my endpoints: `500 - JSON.parse: unexpected character at line 1 column 1 of the JSON data`  I using prisma to connect to a simple Sqlite file-based db.

My vercel build script: 
`"vercel-build": "prisma generate && prisma migrate deploy && vite build",`

Also, fwiw i have added the following line to the package.json that i've found in other issues:
`"postbuild": "cp node_modules/@prisma/engines/*query* .vercel_build_output/functions/node/render/; cp prisma/schema.prisma .vercel_build_output/functions/node/render/; ls .vercel_build_output/functions/node/render/"`

**Comment by austincrim** [comment, marked as answer]
Hey @bmccorm2 👋,

It would be really helpful if you could include a minimal reproduction repo of your issue so I can ensure that I can accurately recreate it with your setup.

**Reply by bmccorm2** [reply]
https://github.com/bmccorm2/issue-deploying-sveltekit-vercel-prisma.git

I think i am missing something simple and just can't find it in the documentation.  It is a fairly vanilla build; i've changed these two things:

- build script: `prisma generate && prisma migrate deploy && vite build`
- init script: `pnpm i`
- env variable for the databaseUrl (just a sqlite db)

**Reply by austincrim** [reply]
I don't get the same error message but a different one regarding opening the SQLite database. It's not possible to use SQLite on Vercel because it doesn't have persistent storage.

Do you get the 500 error when running locally or after deploying?

**Reply by bmccorm2** [reply]
Ahhh that makes sense...that is my problem.  Thank you!

A quick aside, can you recommend any db hosting platforms?

**Reply by austincrim** [reply]
[Railway](https://railway.app), [Render](https://render.com), [Planetscale](https://planetscale.com), and [Supabase](https://supabase.com) are all great, free options, depending on your needs!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
